### PR TITLE
Merge sync e4cd4e6

### PR DESCRIPTION
**Pull request by dkstyle**
## Summary 

Sync to latest upstream [#e4cd4e6](https://github.com/webpack/webpack.js.org/commit/e4cd4e6)

### CODE REVIEW

File: src/content/configuration/cache.mdx [hunk 1]
```diff
@@ -112,6 +113,25 @@ module.exports = {
 };
 ```
 
+### cache.cacheUnaffected
+
+변경되지 않은 모듈의 캐시를 계산하고 변경되지 않은 모듈만 참조합니다. 이는 [`cache.type`](#cachetype)이 `'memory'`로 설정된 경우에만 사용할 수 있으며, [`experiments.cacheUnaffected`](/configuration/experiments/#experimentscacheunaffected)가 활성화 되어 있어야 사용할 수 있습니다.
```
Comment (choilim) [inline src/content/configuration/cache.mdx]: 띄어쓰기: `활성화 되어` --> `활성화되어`

File: src/content/configuration/cache.mdx [hunk 2]
```diff
@@ -302,6 +322,25 @@ module.exports = {
 };
 ```
 
+### cache.memoryCacheUnaffected
+
+변경되지 않은 모듈의 캐시를 계산하고 변경되지 않은 모듈만 참조합니다. 이는 [`cache.type`](#cachetype)이 `'filesystem'`으로 설정된 경우에만 사용할 수 있으며, [`experiments.cacheUnaffected`](/configuration/experiments/#experimentscacheunaffected)가 활성화 되어 있어야 사용할 수 있습니다.
```
Comment (choilim) [inline src/content/configuration/cache.mdx]: 띄어쓰기: `활성화 되어` --> `활성화되어`

File: src/content/configuration/stats.mdx [hunk 3]
```diff
@@ -297,6 +312,21 @@ module.exports = {
 };
 ```
 
+### stats.groupAssetsByPath
+
+`boolean`
+
+애셋 결로별로 애셋을 그룹화할지 여부를 `stats`에 알려줍니다.
```
Comment (choilim) [inline src/content/configuration/stats.mdx]: 혹시 `애셋 경로별로` 일까요??

Comment (dkstyle) [reply src/content/configuration/stats.mdx]: 아 맞아요...ㅠㅠ

**Review by choilim — APPROVED** [review]

**Review by YukJiSoo — APPROVED** [review]

**Review by keipark — APPROVED** [review]

**Pull request merged**